File: FoodDelivery.DAL/EF/Entities/RestaurantRequest.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace FoodDelivery.DAL.EF.Entities
{
	public class RestaurantRequest
	{
		[Key]
		public Guid Id { get; set; }

		[ForeignKey("UserProfile")]
		public Guid UserProfileId { get; set; }

		public int Status { get; set; }

		[ForeignKey("Restaurant")]
		public Guid RestaurantId { get; set; }

		public DateTime CreatedDate { get; set; }

		public DateTime? ClosedDate { get; set; }

		public virtual UserProfile UserProfile { get; set; }
		public virtual Restaurant Restaurant { get; set; }
	}
}

File: FoodDelivery.Web/src/app/models/restaurantRequest/RestaurantRequest.ts
import { RestaurantType } from '../restaurant/RestaurantType';
import { UserShortProfile } from '../userProfile/UserShortProfile';
import { Guid } from 'guid-typescript';

export class RestaurantRequest{
    public id?: Guid;
    public name: string;
    public userProfile: UserShortProfile;
    public status: number;
    public createdDate: Date;
    public closedDate: Date
    public type: RestaurantType;
}